codecov.yaml

```diff
@@ -3,3 +3,6 @@ coverage:
     project:
       default:
         informational: true
+    patch:
```

Comment:
Why is this needed?

Reply:
Fix the codecov CI for `codecov/patch` to not fail the CI